### PR TITLE
Fix markdownlint issues

### PR DESCRIPTION
**Pull request by vbossica**
Simply fix the markdownlint warnings

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,39 +4,39 @@ Thank you for your interest in contributing to the Azure DevOps Extension!
...
 👍 - upvote
 👎 - downvote
 
-#### Submit New Feature request
+### Submit New Feature request
 
 Use our Feature Request [template](https://github.com/Microsoft/azure-devops-cli-extension/issues/new/choose) to create a new feature request.
 
-# Get started
+## Get started
```
Comment (atbagga) [inline CONTRIBUTING.md]: All the content in this section is now at one level above that it should be. Please lower the heading hierarchy for all subitems in this section like Pull request etiquette etc.

Comment (atbagga) [reply CONTRIBUTING.md]: Do double check that out in other files as well. 
Due to pushing each top heading some  subitems have come to same level as the section heading.

**Comment by gauravsaralMs** [comment]
If possible can we also add a check in PR pipeline (YAML) to make sure this does not break again?
cc: @atbagga

File: CONTRIBUTING.md (hunk 1 above)
Comment (vbossica) [reply CONTRIBUTING.md]: Double-checked

**Comment by atbagga** [comment]
> If possible can we also add a check in PR pipeline (YAML) to make sure this does not break again?
> cc: @atbagga

Created a task - not blocking the PR for this. #387

**Pull request merged**